### ISSUE TITLE
Creating a massive scale IoT project

### ISSUE THREAD

**Issue by TopperKain** [renodevelopers/meetups#76]
I just got engaged on a pretty massive IoT project. Phase 1 will wrap up in the May timeframe, so I can could present on (roughly) what we built, how it was built, and the tools used in the June timeframe.

**Comment by w1zeman1p** [comment]
Sounds awesome!

How about a 45 min talk on July 25? If that works for you shoot me an email with your bio: cjavilla@gmail.com

**Comment by TopperKain** [comment]
Hi,
I need to reschedule my presentation to later, I'm now going to be out of town on July 25 for sure.

**Comment by Seberius** [comment]
Hey @TopperKain, would you still like to do this talk?  I think it would be an interesting topic.

**Comment by hekaldama** [comment]
+1

**Comment by TopperKain** [comment]
Yes, sorry I didn't see this till now. Can you propose a time and I'll (properly) block it off?>

**Comment by Seberius** [comment]
Hey @TopperKain , we have Dec 19 and Jan 23 open for talks, would either of those work for you?

**Comment by TopperKain** [comment]
December 19th should be great! Very unlikely anyone will make me travel the week of Christmas :)

**Comment by hekaldama** [comment]
Great! Excited. Thanks @TopperKain

**Comment by Seberius** [comment]
You are set for Dec 19.  Looking forward to the talk.

**Comment by Seberius** [comment]
Hey @TopperKain , just a reminder for this Monday. I am looking forward to the talk! I will be at the Reno Collective by ~5:30. Let me know if you need anything setup ahead of time.

**Comment by TopperKain** [comment]
I'll be there. I'm going to be all slides all the time.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by TopperKain** [comment]
Hit send too soon, I meant to say it's going to all slides so no special set up other than the projector.

**Comment by Seberius** [comment]
Roger that, see you there.
